sys/phydat/phydat_str.c

```diff
@@ -132,6 +132,18 @@ const char *phydat_unit_to_str(uint8_t unit)
     }
 }
 
+const char *phydat_unit_to_str_verbose(uint8_t unit)
+{
+    switch (unit) {
+        case UNIT_UNDEF:    return "undefined";
+        case UNIT_NONE:     return "none";
+        case UNIT_BOOL:     return "bool";
```

Comment:
Given that the type for booleans is already defined in JSON, would it make sense to save some bytes here and also map `UNIT_BOOL` this to `"none"` here?